.github/CODEOWNERS

```diff
@@ -0,0 +1,14 @@
+# This file defines the individuals or teams that are responsible for code in this repository.
+# Each line is a file pattern followed by one or more owners.
+# Reference: https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# below owners will be requested for review when someone opens a pull request.
+*       @pilor @robga
```

Comment:
@pilor FYI - I do not know who are all the owners of this repo. Just picked Robert and you.